br/pkg/lightning/common/common.go

```diff
@@ -39,6 +39,7 @@ var DefaultImportantVariables = map[string]string{
 	"default_week_format":     "0",
 	"block_encryption_mode":   "aes-128-ecb",
 	"group_concat_max_len":    "1024",
+	"tidb_backoff_weight":     "6",
```

Comment:
not sure why the default value is not same as tikvstore.DefBackOffWeight

Reply:
we want to increase the default value for lightning

Reply:
Can you explain why we want to increase it?